lib/LTP/WhiteList.pm

```diff
@@ -129,6 +140,10 @@ sub find_whitelist_testsuite {
     return undef unless defined($path) and -e $path;
 
     my $content = path($path)->slurp;
+    my (undef, undef, $suffix) = fileparse($path, '\.[^\.]*');
+    if ($suffix ne '.json') {
+        $content = yaml2json($content);
```

Comment:
Is there a reason to load native data from YAML, then convert it to a JSON string, and then decode this string to native data again two lines later?
Wouldn't `$content = YAML::PP::Load(decode_utf8 $content)` or even `$content = YAML::PP::LoadFile($path)` be enough?